cpp/src/link_analysis/pagerank.cu

```diff
@@ -355,15 +355,19 @@ void pagerank(raft::handle_t const &handle,
   CUGRAPH_EXPECTS(pagerank != nullptr, "Invalid API parameter: Pagerank array should be of size V");
   // Multi-GPU
   if (handle.comms_initialized()) {
-    CUGRAPH_EXPECTS(personalization_subset == nullptr,
-                    "Invalid API parameter: Multi-GPU Pagerank does not support Personalized "
-                    "variant currently, please use the single GPU version for this feature. The "
-                    "multi-GPU support is comming.");
     CUGRAPH_EXPECTS(has_guess == false,
                     "Invalid API parameter: Multi-GPU Pagerank does not guess, please use the "
                     "single GPU version for this feature");
     CUGRAPH_EXPECTS(max_iter > 0, "The number of iteration must be positive");
-    cugraph::opg::pagerank<VT, ET, WT>(handle, graph, pagerank, alpha, max_iter, tolerance);
+    cugraph::opg::pagerank<VT, ET, WT>(handle,
```

Comment:
Are the opg to mg changes only being done in the documentation and not code? One way to make incremental fix without changing the whole code would be to do this :


```
namespace cugraph
{
namespace opg
{
void my_func(void) {}
}
namespace mg = opg;
}

void calling_func(void) {
  cugraph::mg::my_func();
}

```

Reply:
I updated the whole codebase to remove `opg` occurrences